examples/cbortest/Makefile

```diff
@@ -0,0 +1,34 @@
+############################################################################
+# apps/examples/cbortest/Makefile
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+############################################################################
+
+include $(APPDIR)/Make.defs
+
+# TINYCBOR_TEST built-in application info
+
+PROGNAME = $(CONFIG_EXAMPLES_TINYCBOR_TEST_PROGNAME)
+PRIORITY = $(CONFIG_EXAMPLES_TINYCBOR_TEST_PRIORITY)
+STACKSIZE = $(CONFIG_EXAMPLES_TINYCBOR_TEST_STACKSIZE)
+MODULE = $(CONFIG_EXAMPLES_TINYCBOR_TEST)
+
+# TINYCBOR_TEST Example
+CFLAGS += -I$(APPDIR)$(DELIM)fsutils$(DELIM)libtinycbor
```

Comment:
don't need

Reply:
It is required, without it we get this error:
```
./cbortest_main.c:33:10: fatal error: tinycbor/cbor.h: No such file or directory
 #include "tinycbor/cbor.h"
          ^~~~~~~~~~~~~~~~~
```

Reply:
but you already add it to the search path in Make.defs:
https://github.com/apache/nuttx-apps/pull/1725/files#diff-09f0f42da4d976649c51cc9cba5249554b707d76c1d6aee55c6e7bad598988c9R23

Reply:
That one is not taking effect, I will remove this one so